Assets/Sqripts/Menu.cs

```diff
@@ -0,0 +1,22 @@
+﻿using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class Menu : MonoBehaviour
+{
+    [SerializeField] private Animator _authors;
+
+    public void Play()
+    {
+        SceneManager.LoadScene("SampleScene");
```

Comment:
https://trello.com/c/UAeMGORw/17-%D0%BE%D1%82%D0%BA%D0%B0%D0%B7-%D0%BE%D1%82-%D1%81%D1%82%D1%80%D0%BE%D0%BA%D0%BE%D0%B2%D1%8B%D1%85-%D0%BB%D0%B8%D1%82%D0%B5%D1%80%D0%B5%D0%BB%D0%BE%D0%B2